testem.json

```diff
@@ -3,9 +3,9 @@
   "test_page": "tests/index.html?hidepassed&coverage",
   "disable_watching": true,
   "launch_in_ci": [
-	"Firefox"
+	   "Firefox"
   ],
   "launch_in_dev": [
-	"Chrome"
+	   "Firefox"
```

Comment:
I am curious about this change. I understand that we might want the browser type to be the same for both `dev` and `ci` so we can easily see the test failures but we should also consider that if we do this we are not running tests in `Chrome`.